test/MicroOrm.Dapper.Repositories.Tests/SqlGeneratorTests/MySqlGeneratorTests.cs

```diff
@@ -100,6 +101,16 @@ public void Insert_QuoMarks()
 
             Assert.Equal("INSERT INTO `Addresses` (`Street`, `CityId`) VALUES (@Street, @CityId); SELECT CONVERT(LAST_INSERT_ID(), SIGNED INTEGER) AS `Id`", sqlQuery.GetSql());
         }
+        
+        [Fact]
+        public void Insert_UseKeyAsIdentity_QuoMarks()
+        {
+            MicroOrmConfig.UseKeyAsIdentity = true;
```

Comment:
Seems, that this broke test: https://ci.appveyor.com/project/phnx47/dapper-repositories/build/tests